perl/README.md

```diff
@@ -0,0 +1,1028 @@
+| chr20 | 1       | INS          | A        | +C  | 0    | NA00003         | 0.333          | NA          | 0            | NA           | 0           | 1   | 6   | 0.167     | 1       | SIMPLE_INSERTION            |
+| chr20 | 1       | INS          | A        | +CC | 0    | NA00001;NA00002 | 0.667          | NA00003     | 0.333        | NA           | 0           | 4   | 6   | 0.667     | 1       | INSERTION_BETWEEN_TWO_BASES |
+| chr20 | 1234568 | MULTIALLELIC | T        | -3  | 0    | NA00001         | 0.333          | NA00003     | 0.333        | NA           | 0           | 3   | 6   | 0.5       | 1234567 | microsat1                   |
+| chr20 | 1234568 | MULTIALLELIC | T        | +A  | 0    | NA00002         | 0.333          | NA          | 0            | NA           | 0           | 1   | 6   | 0.167     | 1234567 | microsat1                   |
+| chr20 | 3       | MNP          | C        | A   | 2    | NA              | 0              | NA00003     | 0.5          | NA00001      | 0.333       | 2   | 4   | 0.5       | 3       | EXAMPLE_MISSING_MNP         |
+| chr20 | 4       | MNP          | C        | A   | 2    | NA              | 0              | NA00003     | 0.5          | NA00001      | 0.333       | 2   | 4   | 0.5       | 3       | EXAMPLE_MISSING_MNP         |
+| chr20 | 5       | MNP          | C        | A   | 2    | NA              | 0              | NA00003     | 0.5          | NA00001      | 0.333       | 2   | 4   | 0.5       | 3       | EXAMPLE_MISSING_MNP         |
+
+Expected Bystro Genotype Dosage Matrix Output:
+
+```python
+import pandas as pd
+df = pd.read_feather('example_vcf_dosage_matrix.feather')
+print(df)
+```
+
+| locus              | NA0001 | NA0002 | NA0003 |
+| ------------------ | ------ | ------ | ------ |
+| chr20:1:A:T        | 1      | 0      | 0      |
+| chr20:1110696:A:G  | 1      | 1      | 0      |
+| chr20:1110696:A:T  | 1      | 1      | 2      |
+| chr20:1:A:+C       | 0      | 0      | 1      |
+| chr20:1:A:+CC      | 1      | 1      | 2      |
+| chr20:1234568:T:-3 | 1      | 0      | 2      |
+| chr20:1234568:T:+A | 0      | 1      | 0      |
+| chr20:3:C:A        | -1     | 0      | 2      |
+| chr20:4:C:A        | -1     | 0      | 2      |
+| chr20:5:C:A        | -1     | 0      | 2      |
+
+- Note that missing genotypes are reprsented as -1 in the genotype dosage matrix output
+- If a sample's genotype contains any missing genotypes, the sample is considered missing for the site
+
+#### Explanation for SIMPLE_SNP
+
+VCF Representation:
+
+| CHROM | POS | ID         | REF | ALT | QUAL | FILTER | INFO | FORMAT      | NA00001        | NA00002        | NA00003        |
+| ----- | --- | ---------- | --- | --- | ---- | ------ | ---- | ----------- | -------------- | -------------- | -------------- |
+| 20    | 1   | SIMPLE_SNP | A   | T   | 50   | PASS   | .    | GT:GQ:DP:HQ | 0/1:54:7:56,60 | 0/0:48:4:51,51 | 0/0:48:4:51,51 |
+
+Bystro Representation:
```

Comment:
Recommend some kind of way to show that this is just a representative example, something like either one below.
```suggestion
Bystro Representation (VCF-related fields):
```
```suggestion
Bystro Representation (first 17/150 fields):
```

Reply:
Maybe this makes more sense at line 144

Reply:
I think it is more than 150, 220 for hg38 is what I remember. 

I guess my question is, why is that important?

Reply:
The point of this section is to describe how we handle variants compared to the standard VCF format. The number of annotations isn't that important. I just wanted to mention that this isn't the full output (`see below for the description` of what is in the full output of the default hg38 and hg19 databases), lest they be confused that they only see 17 columns.

Reply:
So that people get a general sense that this is just a small representative example, though I do think it makes more sense to put this up at the description at line 121

Reply:
I would also think "VCF-related" may add confusion, because really all of the annotations are VCF related (the sites come from the VCF file in these examples), and at the same time, only 2 of them are (vcfPos and id are specific to VCF files; SNP files don't have them; all other fields pertain to both). Let's just keep it as is?

Reply:
> So that people get a general sense that this is just a small representative example, though I do think it makes more sense to put this up at the description at line 121

What do you mean by that? This whole section is about how we handle variants vs the VCF spec.

Reply:
Yep I'm ok with keeping the tables as they are. Maybe the top of the fields description is a good spot for the total number? If it's not an impressive sounding metric or number, we can also just skip it in general.

Reply:
Sure, I can add it to that section

Reply:
added